src/luaotfload-main.lua

```diff
@@ -18,7 +18,7 @@ local authors = "\z
     Marcel Krüger\z
 "
 -- version number is used below!
-local ProvidesLuaModule = { 
+local ProvidesLuaModule = {
```

Comment:
While this change generally makes sense, this final space is generally present in all of our files, so we should at some point  we should get rid of it consistently. But that is best done in one central commit when we have no important branches because changing stuff that close to the version number field leads to lots of merge conflicts.